pkgs/tools/audio/unflac/unflac.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/main.go b/main.go
+index 25281e6..48eaa4b 100644
+--- a/main.go
++++ b/main.go
+@@ -32,6 +32,8 @@ var (
+ )
+ 
+ func main() {
++	os.Setenv("PATH","@path@")
```

Comment:
I would prefer [makeWrapper](https://nixos.org/manual/nixpkgs/unstable/#fun-makeWrapper) be used to set $PATH. I don't know of a strict policy on this but I like being able to bypass the shell wrapper if I need to.

Reply:
Thank you for the suggestion! Just amended my commit to use makeWrapper